docs/SITE_DOCS_FIGMA_MAKE.md

```
@@ -281,14 +281,15 @@ D) CUSTOM SCRIPTS & BUNDLES
 
 ### Custom Scripts
 
-GitSpace uses convention-based scripts stored per workspace in `.gitspace/`:
+GitSpace uses convention-based scripts stored per workspace in `.gitspace/scripts/`:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)